src/classes/VOL_CTRL_VolunteersBulkEnterHours_TEST.cls

```diff
@@ -121,7 +121,7 @@ private with sharing class VOL_CTRL_VolunteersBulkEnterHours_TEST {
         // test the save!
         ctrl.SaveVolunteerHours();
         system.debug(ctrl.strSaveResults);
-        System.Assert(ctrl.strSaveResults.startsWith('Error'));
+        System.assert(ctrl.strSaveResults.contains(Label.labelMassEditErrorOnSave), 'An error is expected.');
```

Comment:
@davisagli / @sam-knox - Please let me know what you think of this instead. This way we are still validating that the expected error is returned without hardcoding the value.

Reply:
@jjbennett Looks good to me. Thanks for the improvement.